docs/source/posts/2024/2024-08-15_week_10.rst

```diff
@@ -0,0 +1,39 @@
+Week 10: Learning GraphQL
+=========================
+
+.. post:: Aug 15 2024
```

Comment:
5th of August, or any other day of that week for better sequencing in blog page. As serge mentioned.

Reply:
I gave the dates of when I wrote the posts. The sequencing is fixed with a new commit.

Reply:
Understood. Fine by me then.